packages/v4/src/content/design-guidelines/components/alert/alert.md

```diff
@@ -3,144 +3,171 @@ id: Alert
 section: components
 related: ['Alert', 'Alert group']
 ---
-
-An **alert** is a notification that provides brief information to the user without blocking their workflow.
-
-There are 2 main types of alerts:  
-
-- **Inline alerts**: Inline alerts communicate information about a specific user action on a page. An inline alert appears within the content area and disappears when the user closes it or navigates away from the page.
+ 
+An **alert** is a notification that provides brief information to the user without blocking their workflow.
+ 
+## **Inline alerts**
+### **Bordered inline alerts**
+Bordered inline alerts communicate information about a specific user action on a page. An inline alert appears within the content area and disappears when the user closes it or navigates away from the page.
 
 ![inline notification placement](./img/inline-notification.png)
-
-- **Toast alerts**: Toast alerts communicate information about an update, confirmation, or other activity. A toast alert overlays content in the upper-right corner of the page and disappears when it times out or when the user dismisses it.
-
+ 
+### **Plain inline alerts**
```

Comment:
Make sure all the headers are NOT bolded! using the hashtags is enough for headers!